electrum/wallet_db.py

```diff
@@ -129,7 +129,7 @@ def get_split_accounts(self):
             data2['suffix'] = 'imported'
             result = [data1, data2]
 
-        elif wallet_type in ['bip44', 'trezor', 'keepkey', 'ledger', 'btchip', 'digitalbitbox', 'safe_t']:
+        elif wallet_type in ['bip44', 'trezor', 'keepkey', 'ledger', 'btchip', 'digitalbitbox', 'safe_t', 'jade']:
```

Comment:
this is not needed, see https://github.com/spesmilo/electrum/commit/cbc69742b958aaacdad9612d7cf1564c32205efb

Reply:
Removed.